src/Utils/FilterFunctions/filterByRating.js

```diff
@@ -23,8 +24,10 @@ const filterByRating = (filteredArray, state) => {
   if (state.byRating.fourStar) {
     tempArray = withoutFiltered.filter((prod) => prod.rating >= 4);
   }
+  if (state.byRating.fiveStar) {
+    tempArray = withoutFiltered.filter((prod) => prod.rating >= 4.8);
+  }
```

Comment:
Suggestion : could have done this without using a temp Array , return the destructured withoutFiltered Array

Reply:
Ok, Noted